### PR TITLE
[CHORE] Update view in applayout

### PR DESCRIPTION
**Pull request by VictorTavaress**
### Description
in this PR I add elipses in sidebar title, subtitle and headline

### Screenshots
image

### CODE REVIEW

File: src/components/Layout/AppLayout/AppLayout.vue [hunk 1]
```diff
@@ -275,10 +332,10 @@ $sidebar-width-collapsed: 60px;
...
   }
 
-    &:has(.pb-hint) {
-      overflow: visible;
-      padding-right: 15px;
-    }
+    // &:has(.pb-hint) {
+    //   overflow: visible;
+    //   padding-right: 15px;
+    // }
```
Comment (gabrielforster) [inline src/components/Layout/AppLayout/AppLayout.vue:338]: **polish:** 
```suggestion
```

File: src/components/Layout/AppLayout/AppLayout.vue [hunk 2]
```diff
@@ -275,10 +332,10 @@ $sidebar-width-collapsed: 60px;
...
```
Comment (CauaKath) [inline src/components/Layout/AppLayout/AppLayout.vue:339]:
```suggestion
```

**Review by CauaKath — CHANGES_REQUESTED** [review]

File: src/components/Layout/AppLayout/AppLayout.vue [hunk 3]
```diff
@@ -209,9 +257,18 @@ export default {
...
+      this.headerSizeHeight = header ? header.offsetHeight : 0;
+      this.$nextTick(() => {
+        setTimeout(() => {
+          const { title, subtitle, headline } = this.$refs;
+
+          this.titleSizeWidth = title.offsetWidth < title.scrollWidth;
+          this.subtitleSizeWidth = subtitle.offsetWidth < subtitle.scrollWidth;
+          this.headLineSizeWidth = headline.offsetWidth < headline.scrollWidth;
+        }, 5000);
+      });
```
Comment (gabrielforster) [inline src/components/Layout/AppLayout/AppLayout.vue:269]: **note:** How does this 5 seconds delay timeout work? Can u record a video on it?

**Review by gabrielforster — CHANGES_REQUESTED** [review]

File: src/components/Layout/AppLayout/AppLayout.vue (hunk 3 above)
Comment (VictorTavaress) [reply src/components/Layout/AppLayout/AppLayout.vue:269]: This delay is just for the component to update correctly when it is waiting for the sidebar content that comes through requests. This delay is almost imperceptible.

**Review by CauaKath — APPROVED** [review]
LGTM

**Review by gabrielforster — APPROVED** [review]

**Pull request merged**